src/handlers/hello-world.ts

```diff
@@ -32,12 +32,18 @@ export async function helloWorld(context: Context) {
   logger.info("Hello, world!");
   logger.debug(`Executing helloWorld:`, { sender, repo, issueNumber, owner });
 
+  let response = configurableResponse;
+  if (customStringsUrl) {
+    const { string } = await fetch(customStringsUrl).then((value) => value.json());
+    response = string;
+  }
+
   try {
     await octokit.issues.createComment({
       owner: payload.repository.owner.login,
       repo: payload.repository.name,
       issue_number: payload.issue.number,
-      body: configurableResponse,
+      body: response,
```

Comment:
Keep `configurableResponse` untouched, it should serve as an example for config parameters.

We should also show a real usage of the `customStringsUrl` strings customization. So you may add a new command or respond one extra time but with a string from the `json` file set in `customStringsUrl` config param.

Reply:
@rndquu I assume the format could be like it, what do you think? 

```json
{
  "string": "A custom string."
}
```

Reply:
>  I assume the format could be like it, what do you think?

Yes, the format could be (example):
```json
{
  "greeting": "Hello user",
  "taskAssignment": "You've been assigned to the task"
}
```